dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -71,14 +70,14 @@ public CultureInfo Culture
         this._serviceSelector;
 
     /// <summary>
-    /// Gets the <see cref="IDelegatingHandlerFactory"/> to use when constructing <see cref="HttpClient"/>
-    /// instances for use in HTTP requests.
+    /// Gets the <see cref="HttpClient"/> to use for HTTP requests.
     /// </summary>
     /// <remarks>
-    /// This is typically only used as part of creating plugins and functions, as that is typically
-    /// when such clients are constructed.
+    /// This is typically only used as part of creating services, plugins, and functions, as that is typically
+    /// when such clients are constructed. This instance may be ignored if an <see cref="HttpClient"/> was provided
+    /// to the relevant component directly or via other means.
     /// </remarks>
-    public IDelegatingHandlerFactory HttpHandlerFactory { get; }
+    public HttpClient? HttpClient { get; }
```

Comment:
What happens when we have different services with different clients? Shoudn't the HttpClient be provided at the service configuration not in the Kernel?